src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,19 @@
+import { User } from '../../types/User';
+
 // Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
```suggestion
  const { name, email } = user;
```
you can left them on the same line if there are less then 3 values